pxtsim/runtime.ts

```diff
@@ -37,6 +37,15 @@ namespace pxsim {
             return Date.now();
         }
 
+        export function nowUs(): number {
+            return (performance as any).now() ||
```

Comment:
You need to do `if (typeof performance != "undefined")` ... for node.js

For node you could also do `process.hrtime()`